examples/jenkins/pipeline/README.md

```diff
@@ -39,9 +39,10 @@ jenkins template represented by jenkinstemplate.json by running these commands a
 
         $ oc new-app jenkins-ephemeral
 
-    Make a note of the Jenkins password reported by new-app.
+    Note: in previous versions, you were asked to make a note of the Jenkins password reported by new-app.  However, now when using either of the default templates, you can supply any string as the password, but
+    OpenShift by default uses the `Allow All` identity provider.
```

Comment:
no need to talk about what we used to tell people to do.  just describe what they need to do/how it works now.